_dev/src/store/modules/accounts/actions.ts

```diff
@@ -125,6 +125,9 @@ export default {
         return;
       }
       try {
+        const {token} = await dispatch(ActionsTypes.REQUEST_SITE_VERIFICATION_TOKEN, correlationId);
+        dispatch(ActionsTypes.SAVE_WEBSITE_VERIFICATION_META, token);
```

Comment:
I'm expecting this to be done at all times, even if the website is already claimed and verified